packages/sdk/src/services/AgentService.ts

```diff
@@ -28,10 +28,27 @@ export class AgentService {
    * @param {number} servicePrice - The price of the service.
    * @returns {Promise<string>} A promise that resolves to the agent address.
    */
-  async registerAgent(address: string, name: string, uri: string, serviceName: string, servicePrice: number): Promise<boolean> {
+  async registerAgent(
```

Comment:
discussed this with @LilFatFrank 
there's two options here - 1. upload metadata on the frontend side, or 2. provide an object with any metadata as an argument that the SDK will deploy to IPFS.
I think right now option is the right now, or this will be a mess